src/store/index.ts

```diff
@@ -155,22 +155,40 @@ export default new Vuex.Store({
       },
     ],
     user: {
-      title: "Ms.",
-      given_name: "Maria",
-      family_name: "Missionowner",
-      email: "maria.missionowner-civ@mail.mil",
-      phone_number: "(555)-555-5555",
-      service_branch: "U.S. Army",
-      citizenship: "United States",
-      dod_id: "1234567890",
-      designation: "Civilian",
+      title: "",
+      given_name: "",
+      family_name: "",
+      email: "",
+      phone_number: "",
+      service_branch: "",
+      citizenship: "",
+      dod_id: "",
+      designation: "",
     },
     validationStamp: {},
   },
   mutations: {
     changeLoginStatus(state, status: boolean) {
       state.loginStatus = status;
     },
+    changeUser(state, user: any) {
+      // These attributes will come across directly and cleanly from the
+      // u[stream identity provider and Cognito
+      state.user.given_name = user?.given_name ?? "";
+      state.user.family_name = user?.family_name ?? "";
+      state.user.email = user?.email ?? "Not Provided";
+      // This field will have to be a custom Cognito attribute and so
+      // the source object may have a different format.
+      state.user.dod_id = user?.["custom:dod_id"] ?? "1234567890";
+      state.user.citizenship = user?.["custom:citizenship"] ?? "United States";
+      state.user.designation = user?.["custom:designation"] ?? "Civilian";
+      // This field may not be available from our identity provider
+      state.user.phone_number = user?.phone ?? "(555) 555-5555";
+      // There is not currently a known way to get this information from
+      // the identity provider.
+      state.user.service_branch = "U.S. Army";
+      state.user.title = "Ms.";
+    },
```

Comment:
I think my changes in this file are the most bad of my changes and the ones where I really didn't know if this was the right way to do it.